tests/model/potential/test_bond.py

```diff
@@ -0,0 +1,679 @@
+"""Unit tests for bond module."""
+
+import tempfile
+import unittest
+
+import numpy
+
+import relentless
+
+
+class test_BondParameters(unittest.TestCase):
```

Comment:
Since `BondParameters` is a pass through class, it is not necessary to duplicate a test that exists elsewhere (I'm assuming this is copied, yes?)